spec/PeriodicBackgroundSync-index.bs

```diff
@@ -107,36 +107,39 @@ A <dfn>state</dfn>, which is one of "`pending`", "`firing`", "`suspended`" or "`
+* <dfn>minimum periodic sync interval across origins</dfn>, an unsigned long long, that represents the minimum gap between [=periodicsync event|periodicsync events=] across all origins in milliseconds.
+
+These constants are used to mitigate the concerns raised in [[#privacy]], [[#security]] and [[#resources]].
 
 [=minimum periodic sync interval across origins=] MUST be greater than or equal to [=minimum periodic sync interval for any origin=].
-If undefined, these are set to 43200000, which is twelve hours in milliseconds.
+[=minimum periodic sync interval across origins=] SHOULD be greater than or equal to 43200000, which is twelve hours in milliseconds.
 
-Note: Two caps on frequency are needed because conforming to the cap enforced by [=minimum periodic sync interval for any origin=] for each origin can still cause the browser to fire [=periodicsync events=] very frequently. This can happen, for instance, when there are many [=periodic sync registration|periodic sync registrations=] for different origins. [=minimum periodic sync interval across origins=] ensures there's a global cap on how often these events are fired.
+Note: A minimum value of of 43200000 (which is twelve hours in milliseconds) for [=minimum periodic sync interval across origins=] is suggested to mitigate the concerns described in [[#privacy]] and [[#security]].
+
+Note: Two caps on frequency are needed because conforming to the cap enforced by [=minimum periodic sync interval for any origin=] for each [=origin=] can still cause the browser to fire [=periodicsync events=] very frequently. This can happen, for instance, when there are many [=periodic sync registration|periodic sync registrations=] for different origins. [=minimum periodic sync interval across origins=] ensures there's a global cap on how often these events are fired.
 
 The user agent MAY define a <dfn>maximum number of retries</dfn>, a number, allowed for each [=periodicsync event=]. In choosing this, the user agent SHOULD ensure that the time needed to attempt the [=maximum number of retries=] is an order of magnitude smaller than the [=minimum periodic sync interval for any origin=]. If undefined, this number is zero.
 
 Privacy Considerations {#privacy}
 ==================================
 
 ## Permission ## {#permission}
-Periodic Background Sync is only available if the {{PermissionState}} for a {{PermissionDescriptor}} with {{PermissionDescriptor/name}} `"periodic-background-sync"` is {{PermissionState/granted}}. In addition, user agents SHOULD offer a way for the user to disable Periodic Background Sync.
+Periodic Background Sync is only available if the {{PermissionState}} for a {{PermissionDescriptor}} with {{PermissionDescriptor/name}} `"periodic-background-sync"` is {{PermissionState/granted}}. The user agent MAY show UI to the user to ask for this permission. In addition, user agents MUST offer a way for the user to disable Periodic Background Sync.
```

Comment:
Do we need to add a step somewhere for this permission UI to appear (as in, one of the steps in `register`)?

Reply:
register() does have a check for this permission. Do you think we need to note there again that this may show a UI? Permission is granted once per origin, so each register() won't need to prompt the user anyway.

Reply:
Checking permission is one thing, but asking for it is another. If we want browsers to be able to ask for permission, we should be clear about when they can do this.

I guess it'd be the first step of `register()`: The user agent may [request permission to use](https://w3c.github.io/permissions/#request-permission-to-use) `PermissionName` `"periodic-background-sync"`.

Reply:
Makes sense. Added.